benches/files.rs

```diff
@@ -12,7 +12,7 @@ use vector::{
 };
 
 fn benchmark_files_without_partitions(c: &mut Criterion) {
-    let num_lines: usize = 100_000;
+    let num_lines: usize = 10_000;
```

Comment:
Similarly, how would you feel about reducing this down? Criterion is already running in a fairly hot loop and this is a very coarse grained measurement.